apps/spruce/src/pages/waterfall/DateFilter/index.tsx

```diff
@@ -38,6 +40,8 @@ export const DateFilter = () => {
       data-cy="date-picker"
       label="Go to Date"
       max={new Date()}
+      // Testing environments should not have a minimum date restriction due to static test data.
+      {...(isProduction() && { min: subDays(subYears(new Date(), 1), 1) })}
```

Comment:
nit. It might be a little bit more readable using the `sub` function. I think you can do something like this:
```suggestion
      min={isProduction() ? sub(today, { years: 1 }) : undefined}
```

where `today = new Date()` (could be declared as a constant at the bottom of the file if you want)